src/index.js

```diff
@@ -1,117 +1,93 @@
+const isObj = o => typeof o === 'object'
+
+const each = (obj, fun) => {
```

Comment:
This is great ! 

But actually using `Object.entries` already works for both Object & Arrays, so ultimately I didn't use this method in the merged code but I definitely was inspiring 😄 .

Reply:
Wouldn't using ``.entries`` be slower overall than just using .forEach with arrays? I'm probably just nit picky but it could be a concern. Anyhoo that's great if it works with just plain ``.entries`` then simplify   on, no troubles there.